Source/Krypton Toolkit Examples/KryptonMessageBox Examples/Form1.cs

```diff
@@ -138,8 +138,8 @@ private void buttonShow_Click(object sender, EventArgs e)
             }
             else
             {
-                MessageBox.Show(textBoxMessage.Text, textBoxCaption.Text, _mbButtons, _mbIcon,
-                    MessageBoxDefaultButton.Button1, _options);
+                //? Why is this here? MessageBox.Show(textBoxMessage.Text, textBoxCaption.Text, _mbButtons, _mbIcon, MessageBoxDefaultButton.Button1, _options);
```

Comment:
To demonstrate the "Normal" window box, and ensure that the Krypton box follows the same pattern for the same settings.
Especially useful for RTL type testing.

Reply:
Reverted